macOS/DuckDuckGo/DataImport/Bookmarks/FavoritesImportProcessor.swift

```diff
@@ -64,7 +62,7 @@ enum FavoritesImportProcessor {
         processMaybeFolder(&bookmarks.topLevelFolders.syncedBookmarks)
 
         // Create bookmarks objects for favorite shortcuts that don't have matching bookmarks and add them to bookmark bar
-        let uniqueShortcuts = favorites.filter { !foundFavoriteUrls.contains($0.url?.nakedString ?? "") }
+        let uniqueShortcuts = favorites.filter { favoriteUrlIndices.keys.contains($0.url?.nakedString ?? "") }
```

Comment:
I‘m not sure in Swift implementation details right here, but at a glance it seems raising the algo complexity, I suggest changing it to:
`favoriteUrlIndices[$0.url?.nakedString ?? ""] != nil`
this is shorter as well